trial-config-service/src/main/java/uk/ac/ox/ndph/mts/trialconfigservice/config/WebConfig.java

```diff
@@ -0,0 +1,91 @@
+package uk.ac.ox.ndph.mts.trialconfigservice.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.cors.CorsConfiguration;
+import org.springframework.web.cors.UrlBasedCorsConfigurationSource;
+import org.springframework.web.filter.CorsFilter;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import springfox.documentation.builders.*;
+import springfox.documentation.service.*;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spi.service.contexts.SecurityContext;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger.web.SecurityConfiguration;
+import springfox.documentation.swagger.web.SecurityConfigurationBuilder;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.Arrays.asList;
+
+@Configuration
+public class WebConfig implements WebMvcConfigurer {
+    @Value("${spring.security.oauth2.resourceserver.opaquetoken.client-secret}")
+    private String CLIENT_SECRET;
+
+    @Value("${spring.security.oauth2.resourceserver.opaquetoken.client-id}")
+    private String CLIENT_ID;
+
+    @Value("${swagger.authserver.url}")
+    private String AUTH_SERVER;
+
+    private List<AuthorizationScope> authorizationScopeList = new ArrayList<>();
+
+    @Bean
+    public Docket api() {
+        authorizationScopeList.add(new AuthorizationScope("api://mts-apps/admin", "admin scope"));
```

Comment:
Please extract hard coded strings into static members, or even better, to environment variables, or other sort or configurable settings.

Reply:
Moved to statics